composer.json

```diff
@@ -6,22 +6,23 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/allowed_formats": "~1.1",
-        "drupal/core": "~8.6@alpha",
-        "drupal/paragraphs": "~1.2",
+        "drupal/allowed_formats": "^1.1",
+        "drupal/core": "^8.6",
+        "drupal/paragraphs": "^1.3",
         "php": "^7.1"
     },
     "require-dev": {
-        "composer/installers": "^1.2",
-        "cweagans/composer-patches": "~1.0",
-        "drupal-composer/drupal-scaffold": "^2.2",
+        "composer/installers": "~1.5",
+        "cweagans/composer-patches": "~1.6",
+        "drupal-composer/drupal-scaffold": "~2.2",
+        "drupal/coder": "~8.2.12",
```

Comment:
This dependency on Coder is incorrect and should not be here in the first place. Using the `~` operator will only allow the lowest version number to update, so this will prevent to update to Coder 8.3.0 which was recently released.

We don't need this anyway since we do not directly depend on the Coder module for development. This will be pulled in automatically through our dependence on `openeuropa/code-review`.

So this line should just be removed.